package.json

```diff
@@ -105,6 +105,7 @@
     "@swc/core": "1.3.41",
     "@types/app-root-path": "1.2.4",
     "@types/cli": "0.11.21",
+    "@types/lru-cache": "^7.10.10",
```

Comment:
I think this and `"lru-cache"` should be in engine's `package.json`

Reply:
Done